package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.3.1",
   "description": "Validates strings as commit messages",
   "main": "index.js",
+  "bin": {"commitplease": "./commit-msg-hook.js"},
```

Comment:
Normally, you would comment on some code that needs changing. Once the change gets force-pushed, github marks the comment as "a comment on an outdated diff", so it is clear that the comment has been addressed.

Now, I have changed the readme accordingly but did not change package.json, so force-push does not touch this file. Do I write a comment like "thanks, fixed" here, or do I not respond at all?